src/main/frontend/handler/paste.cljs

```diff
@@ -182,38 +182,50 @@
     (paste-copied-blocks-or-text text e html)))
 
 (defn editor-on-paste!
-  ([id]
-   (editor-on-paste! id false))
-  ([id raw-paste?]
-   (fn [e]
-     (state/set-state! :editor/on-paste? true)
-     (let [input (state/get-input)]
-       (if raw-paste?
```

Comment:
I didn't change logic in the formatted paste. Just moved raw-paste? to its own fn